### PR TITLE
Todo manage

### PR DESCRIPTION
**Pull request by CostircoNazar**

### CODE REVIEW

File: src/AddTodo/AddTodo.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    const [title, setTitle] = useState('')
+
+    const createNewTodoItem=(event)=>{
+        if (title.length > 0){
+            const item = {
+                title: title,
+                done: false,
+                filtered: true
+            }
+            addNewTodo(id,item)
```
Comment (HrudakovSerhii) [inline src/AddTodo/AddTodo.js:17]: Add space before addNewTodo function.

File: src/AddTodo/AddTodo.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            setTitle('')
+        }
+        event.preventDefault();
```
Comment (HrudakovSerhii) [inline src/AddTodo/AddTodo.js:21]: add space before event.preventDefault

File: src/AddTodo/AddTodo.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            <input type='text'
+                   placeholder='Add new note.'
+                   name='addNewNote'
+                   value={title}
+                   onChange={(e)=>setTitle(e.target.value)}
+            />
+
+            <button name='saveNewNote'
+                    onClick={createNewTodoItem}>Save</button>
+
```
Comment (HrudakovSerhii) [inline src/AddTodo/AddTodo.js:35]: Please remove this empty line

File: src/TodoGroup/TodoGroup.js [hunk 4]
```diff
@@ -1,18 +1,31 @@
...
             return <TodoItem key={index}
                              title={item.title}
-                             done={item.done}
+                             done={isDone}
+                             handleChange={handleChane}
+
             />} return null
     })
 
+
```
Comment (HrudakovSerhii) [inline src/TodoGroup/TodoGroup.js:21]: There should be only one line between todoItemsList and return code

File: src/TodoGroup/TodoGroup.js [hunk 5]
```diff
@@ -25,6 +38,14 @@ function TodoGroup({todoList,color,created,title}){
                 {todoItemsList}
             </div>
 
+            <div>
+                <AddTodo addNewTodo={addTodo}
+                        id={id}
+                />
+
```
Comment (HrudakovSerhii) [inline src/TodoGroup/TodoGroup.js]: Please remove empty line

File: src/TodoGroup/TodoGroup.js [hunk 6]
```diff
@@ -25,6 +38,14 @@ function TodoGroup({todoList,color,created,title}){
...
+            </div>
+
+
```
Comment (HrudakovSerhii) [inline src/TodoGroup/TodoGroup.js]: Please remove empty lines

File: src/TodoItem/TodoItem.js [hunk 7]
```diff
@@ -1,16 +1,16 @@
 import React from "react";
 import './styles.css'
 
-const TodoItem=({title, done})=>{
+const TodoItem=({title, done,handleChange})=>{
```
Comment (HrudakovSerhii) [inline src/TodoItem/TodoItem.js]: There should be space after each ',' char

**Review by HrudakovSerhii — COMMENTED** [review]
Please review all comments and make fixes in one commit

File: src/TodoItem/TodoItem.js (hunk 7 above)
Comment (CostircoNazar) [reply src/TodoItem/TodoItem.js]: Done

File: src/TodoGroup/TodoGroup.js (hunk 6 above)
Comment (CostircoNazar) [reply src/TodoGroup/TodoGroup.js]: Removed

File: src/TodoGroup/TodoGroup.js (hunk 5 above)
Comment (CostircoNazar) [reply src/TodoGroup/TodoGroup.js]: Removed

File: src/TodoGroup/TodoGroup.js (hunk 4 above)
Comment (CostircoNazar) [reply src/TodoGroup/TodoGroup.js:21]: I have only only one line

File: src/AddTodo/AddTodo.js (hunk 3 above)
Comment (CostircoNazar) [reply src/AddTodo/AddTodo.js:35]: Line 29 ?

File: src/AddTodo/AddTodo.js (hunk 2 above)
Comment (CostircoNazar) [reply src/AddTodo/AddTodo.js:21]: Done

File: src/AddTodo/AddTodo.js (hunk 1 above)
Comment (CostircoNazar) [reply src/AddTodo/AddTodo.js:17]: Done

File: src/AddTodo/AddTodo.js (hunk 3 above)
Comment (HrudakovSerhii) [reply src/AddTodo/AddTodo.js:35]: Line 34

File: src/TodoGroup/TodoGroup.js (hunk 4 above)
Comment (HrudakovSerhii) [reply src/TodoGroup/TodoGroup.js:21]: return(...) is on 29. todoList.map end function is on 23. So there is 6 lines between the code, correct?

**Pull request merged**